cli/deploy-managed-online-endpoint-mlflow-vnet.sh

```diff
@@ -0,0 +1,50 @@
+set -e
+
```

Comment:
@shivanissambare 
How do we add docker building and pushing to acr here

docker build -t mlflowbyoc:1 ./environment/docker-context
docker tag mlflowbyoc:1 {{acr}}.azurecr.io/mlflowbyoc:1
docker push {{acr}}.azurecr.io/mlflowbyoc:1